dtl/triggers.py

```diff
@@ -60,7 +61,7 @@
     (["stonk", "stonks"], {}, ([stonks], ["📈"])),
     (["shit", "bot"], {"reducer": all}, ([], ["feelsbadman"])),
     (["good", "bot"], {"reducer": all}, ([], ["feelsgoodman"])),
-    (["joe", "cheesesteak", "philly"], {}, ([cheesesteak], [])),
+    (["cheesesteak", "philly"], {}, ([cheesesteak_joe, cheesesteak_don], [])),
```

Comment:
Do you want to add emoji reactions to this? You can put them in the list that's the last argument. Although there's no cheesesteak emoji, it might be worth adding the American flag or other food items.

Example:

https://github.com/MrPickles/dtl/blob/1b4a906e8095dbd6181e2af5f6bce57727402e8d/dtl/triggers.py#L45